assets/scss/components/_button.scss

```diff
@@ -338,6 +320,29 @@ Class                   | Description
 }
 
 //
+// Reset buttons
+//
+.button--reset {
+	@include color-all($C_textPrimary);
+	@include buttonHover(transparent);
+	@include standardBorder(none);
+	border-radius: 0;
+	font-weight: normal;
+	min-height: 0;
+	padding: 0;
+
+	&,
+	&.button--disabled,
+	&.button[disabled] {
+		background: transparent;
+	}
+
+	.inverted & {
```

Comment:
I'm getting a light background color when hovering on a disabled button in an inverted context